aya-obj/src/programs/lsm.rs

```diff
@@ -0,0 +1,21 @@
+//! XDP programs.
```

Comment:
This can be removed entirely since we don't need `LsmAttachType` given that we have separate `Lsm` and `LsmCgroup` types.